Sources/iPad/View Controllers/File/DocumentsViewController_iPad.h

```diff
@@ -30,7 +30,7 @@
 @interface DocumentsViewController_iPad : DocumentsViewController <UIPopoverControllerDelegate,WEPopoverControllerDelegate>
 {    
     WEPopoverController *_actionPopoverController;
-    UIPopoverController *_fileFolderActionsPopoverController;
+    WEPopoverController *_fileFolderActionsPopoverController;
 }
 
```

Comment:
On iPad: replace the UIPopover by WEPopover for a better support in rotation & the same appearance as file actions pop
opver (WEPopover is using for this actions)